scripts/docgen/snippets/extractSnippets.ts

```diff
@@ -169,78 +122,52 @@ const extractSnippetRangesFromFile = (sourceFile: SourceFile): SnipRange[] => {
                 .join(",")}.`
         )
     }
-    return snipRanges
+    return labeledSnippets
+}
+
+const extractTextBetweenLines = (
+    lines: string[],
+    firstLine: number,
+    lastLine: number
+) => {
+    const filteredTextArray = []
+    for (let i = firstLine; i <= lastLine; i++) {
+        if (outputWillInclude(lines[i])) {
+            filteredTextArray.push(lines[i])
+        }
+    }
+    return filteredTextArray.join("\n")
 }
+const filterSnipStatements = (lines: string[]) =>
```

Comment:
Let's rename this to `linesToOutput` or similar to clarify that it converts to a string and calls a more generically named function `outputWillInclude` (if we want to add something else to filter in the future, it will be clearer that it should go here).